.gitignore

```diff
@@ -1,2 +1,3 @@
 .vscode
 build
+**/vinstr_str.h
```

Comment:
Remove this. this generate file need not be part of the source code at all.

Reply:
What I was trying to get at is not to modify the code in the repository at build time. In the current flow what you are doing is to generate the `vinstr_str.cpp` file and add it back to the original repo. Instead of doing this, go ahead and generate the object file out of the `vinstr_str.cpp`, `vinstr_str.o`. Then you can add this object file as a dependency to tilt. That way all the generated files only live inside the build directory and you are not modifying anything on the main source code.